inc/site-health.php

```diff
@@ -805,18 +842,32 @@ function test_compatibility() {
 	}
 
 	if ( ! empty( $manual_fixes ) ) {
+		$description = sprintf(
+			'<p>%s</p>%s',
+			__( 'There are known compatibility issues with your active plugins that require manual fixes.', 'pantheon' ),
+			output_compatibility_status_table( $manual_fixes, false )
+		);
+
+		if ( ! empty( $review_fixes ) ) {
+			$manual_table = output_compatibility_status_table( $manual_fixes, false );
+			$review_table = output_compatibility_status_table( $review_fixes, false, true );
+			$description = sprintf(
+				'<p>%1$s</p>%2$s<p>%3$s</p>%4$s',
+				__( 'There are known compatibility issues with your active plugins that require manual fixes.', 'pantheon' ),
+				$manual_table,
+				__( 'There are known incompatibility issues with the following plugins.', 'pantheon' ),
+				$review_table
+			);
+		}
```

Comment:
```suggestion
		$manual_table = output_compatibility_status_table( $manual_fixes, false );
		$description = sprintf(
			'<p>%s</p>%s',
			__( 'There are known compatibility issues with your active plugins that require manual fixes.', 'pantheon' ),
			$manual_table
		);

		if ( ! empty( $review_fixes ) ) {
			$review_table = output_compatibility_status_table( $review_fixes, false, true );
			$description = sprintf(
				'<p>%1$s</p>%2$s<p>%3$s</p>%4$s',
				__( 'There are known compatibility issues with your active plugins that require manual fixes.', 'pantheon' ),
				$manual_table,
				__( 'There are known incompatibility issues with the following plugins.', 'pantheon' ),
				$review_table
			);
		}
```
you're (potentially) calling `output_compatibility_status_table( $manual_fixes, false )` twice, that could be assigned and re-used.